guides.md

```diff
@@ -28,6 +29,165 @@ our values.
+2. Navigate to "Mailboxes" tab
+3. Click "+Add mailbox" button at top-right of screen
+    - If the namespace is already taken by an alias, ensure it's not in use before deleting it.
+    <details>
+      <summary>View Screenshot</summary>
+      <img alt="Screenshot of Mailcow mailbox tab" src="images/email-mailbox-2.png" />
+    </details>
+4. Fill out the following fields, noting the password:
+    - Username: `[info provided by new user]`
+    - Domain: `hypha.coop`
+    - Full name: `[info provided by new user]`
+    - Quota: `4096`
+    - Password: `[auto-generated]`
+    - Active: `checked` :ballot_box_with_check:
+    <details>
+      <summary>View Screenshot</summary>
+      <img alt="Screenshot of Mailcow mailbox creation popup" src="images/email-mailbox-3.png" />
+    </details>
+
+We'll now log in to confirm access, and set up an initial forwarder to make first-time usage simpler.
+
+5. Open an incognito browser and access webmail login: https://mailninja.aseriesoftubez.com/SOGo/
+6. Log into the account you just created.
+    <details>
+      <summary>View Screenshot</summary>
+      <img alt="Screenshot of webmail login screen" src="images/email-mailbox-4.png" />
+    </details>
+7. Navigate through these menus:  
+    _:gear:(top-left) > Mail menu item > Forward tab_
+8. Check "Forward incoming mail" and enter provided primary email, clicking "Keep a copy".
+    <details>
+      <summary>View Screenshot</summary>
+      <img alt="Screenshot of webmail forwarding settings" src="images/email-mailbox-5.png" />
+    </details>
+9. From your own `@hypha.coop` email, send a welcome email, like this template:
+    >
+    > Hi XXXX --
+    >
+    > Yay! Your Hypha email's set up and forwarding to your personal inbox. People can start emailing you at this address immediately, without any extra effort on your part.
+    >
+    > Your new email (and login): XXX@hypha.coop  
+    > Your password: Sent via Signal to XXX-XXX-XXXX
+    >
+    > Having said that, when you'd like to start _sending_ email as XXX@hypha.coop, you'll need to take some more steps.
+    >
+    > The next steps will depend on how you like to handle email:
+    > - [If you'd like to use it from your existing Gmail account...](#using-gmail)
+    > - [If you'd like to use it from our hosted webmail interface...](#using-the-web-interface)
+    > - [If you'd like to use it from another email client...](l#using-your-existing-email-client)
+    >
+    > Once you're set up, let us know by sending a test email to infra@hypha.coop, ideally with a short emoji-only story. (Keypad mashes a-ok!) We'll send a friendly poke in a week if we haven't heard back.
+    >
+    > Questions? Feedback? Reach out via infra@hypha.coop or [`#hyphacoop-infra:tomesh.net`](https://chat.tomesh.net/#/room/#hyphacoop-infra:tomesh.net) for chat.
+    >
+    > In Solidarity,  
+    > xxxxx
+10. Send a low-entropy Signal message with the previously noted password, for example: `🍄 password: xxxxxxxx`
```

Comment:
"low-entropy Signal message" is jargon-y. What do you mean?

Reply:
also, the mushroom is a little... unclear?  "hypha email password" might over more